integration-test/pom.xml

```diff
@@ -51,6 +53,30 @@
 
     <dependency>
       <groupId>${project.groupId}</groupId>
+      <artifactId>livy-repl_2.10</artifactId>
```

Comment:
This is for not importing repl jars to the classpath.This is a build dependency to make sure repl-2.10 and repl-2.11 projects are built before integration-test starts.